File: seleniumChrome.py
from selenium import webdriver
from selenium.webdriver.chrome.options import Options
from selenium.webdriver.chrome.service import Service

options = Options()
options.add_argument('--headless')
options.add_argument('--no-sandbox')

service = Service('./chromedriver')

browser = webdriver.Chrome(service=service, options=options)
browser.get("https://github.com/developer-onizuka")
browser.save_screenshot('screenshot.png')
browser.quit()

File: seleniumChrome.cs
using System;
using System.IO;
using System.Reflection;
using OpenQA.Selenium;
using OpenQA.Selenium.Chrome;

namespace SeleniumTest
{
    class MainClass
    {
        public static void TakeScreenshot(IWebDriver driver, string saveFile)
        {
            ITakesScreenshot ssdriver = driver as ITakesScreenshot;
            Screenshot screenshot = ssdriver.GetScreenshot();
            screenshot.SaveAsFile(saveFile, ScreenshotImageFormat.Png);
        }

        public static void Main()
        {
            ChromeOptions options = new ChromeOptions();
	    options.AddArgument("--headless");
	    options.AddArgument("--no-sandbox");

	    //Find directory with Chrome Driver
            //Console.WriteLine(Assembly.GetExecutingAssembly().Location);
	    
	    IWebDriver browser = new ChromeDriver(".", options);
            browser.Navigate().GoToUrl(@"https://github.com/developer-onizuka");
	    TakeScreenshot(browser, "./test.png");

            browser.Quit();
        }
    }
}
